envoy-control-runner/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/debug/SnapshotDebugController.kt

```diff
@@ -71,6 +94,7 @@ class SnapshotDebugController(controlPlane: ControlPlane) {
     }
 
     class SnapshotNotFoundException : RuntimeException("snapshot missing")
+    class GlobalSnapshotNotFoundException : RuntimeException("Global snapshot missing")
```

Comment:
Shouldn't we write a handler for this exception, returning 404? Just like `handleSnapshotMissing` below.